pkg/envoy/ingress_translator.go

```diff
@@ -147,6 +149,28 @@ func classFilter(ingresses []v1beta1.Ingress, ingressClass []string) []v1beta1.I
 	return is
 }
 
+func validIngressFilter(ingresses []v1beta1.Ingress) []v1beta1.Ingress {
+	vi := make([]v1beta1.Ingress, 0)
+
+	for _, i := range ingresses {
+		ing := make([]v1.LoadBalancerIngress, 0)
+		for _, j := range i.Status.LoadBalancer.Ingress {
+			if j.Hostname == "" && j.IP == "" {
+				logrus.Debugf(`no hostname or ip for loadbalancer found in ingress config for "%+v" in namespace "%+v"`, i.Name, i.Namespace)
```

Comment:
nitpick but generally I would logs with colons, instead of quotes, 
e.g 
```go
logrus.Debugf("no hostname or ip for loadbalancer found in ingress config for: %v in namespace: %v", i.Name, i.Namespace)
```